src/components/ToggleSwitch/index.tsx

```diff
@@ -11,7 +11,7 @@ import {
   TouchableOpacityProps
 } from 'react-native';
 import dimen from '../../utils/dimen';
-import {normalizeFontSize} from '../../utils/fonts';
+import {fonts, normalizeFontSize} from '../../utils/fonts';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement has been modified to include `fonts` from the `../../utils/fonts` module. This is a good change as it allows for more consistent typography across the application by using predefined font styles.
```diff
- import {normalizeFontSize} from '../../utils/fonts';
+ import {fonts, normalizeFontSize} from '../../utils/fonts';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->